src/pycolmap/scene/image.cc

```diff
@@ -105,6 +105,58 @@ void BindSceneImage(py::module& m) {
            py::overload_cast<camera_t>(&Image::Point2D),
            "point2D_idx"_a,
            "Direct accessor for a point2D.")
+      .def("get_xy",
+           [](const Image& self, std::optional<std::vector<point2D_t>> point_ids = std::nullopt) -> py::array {
+               std::vector<double> xy_coords;
```

Comment:
Better create a `Eigen::MatrixX2d` and return it as is, pybind will handle the conversion to a numpy array.